src/com/sfuentes/OnlyPositive.java

```diff
@@ -0,0 +1,18 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+
+public class OnlyPositive {
+  public ArrayList<Double> filterPositiveNumbers(double[] numbers) {
+
+    ArrayList<Double> positives = new ArrayList<>();
+    int i = 0;
+
+    for (double number : numbers)
+      if (number > 0) {
+        positives.add(number);
+      }
```

Comment:
Here you can remove the curly braces since seems that in all the files you are not using them for one line body

Reply:
Fix it.